content/en/remote_configuration/_index.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Remote Configuration
+aliases:
+- /agent/guide/how_rc_works
+- /agent/guide/how_remote_config_works
+- /agent/remote_config
+further_reading:
+- link: "/security/application_security/how-appsec-works/#built-in-protection"
+  tag: "Documentation"
+  text: "How Application Security Monitoring Works"
+- link: "/dynamic_instrumentation/?tab=configurationyaml#enable-remote-configuration"
+  tag: "Documentation"
+  text: "Dynamic Instrumentation"
+- link: "/security/threats/setup"
+  tag: "Documentation"
+  text: "Setting Up Workload Protection"
+- link: "https://www.datadoghq.com/blog/compliance-governance-transparency-with-datadog-audit-trail/"
+  tag: "Blog"
+  text: "Using Datadog Audit Trail"
+- link: "https://www.datadoghq.com/blog/remote-configuration-for-datadog/"
+  tag: "Blog"
+  text: "Apply real-time updates to Datadog components with Remote Configuration"
+algolia:
+  tags: ['remote config', 'remote configuration']
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Remote Configuration is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
```

Comment:
We need to remember to remove this when launching in Gov (planned for towards the end of the year)